cwms-data-api/src/test/java/cwms/cda/api/MeasurementControllerTestIT.java

```diff
@@ -0,0 +1,654 @@
+import static cwms.cda.security.KeyAccessManager.AUTH_HEADER;
+import fixtures.CwmsDataApiSetupCallback;
+import fixtures.TestAccounts;
+import static io.restassured.RestAssured.given;
+import io.restassured.filter.log.LogDetail;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.sql.SQLException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.http.HttpServletResponse;
+import mil.army.usace.hec.test.database.CwmsDatabaseContainer;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.AfterAll;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.Matchers.*;
+
+@Tag("integration")
+final class MeasurementControllerTestIT extends DataApiTestIT {
+
+    private static final String OFFICE_ID = TestAccounts.KeyUser.SPK_NORMAL.getOperatingOffice();
+    private static final List<Stream> STREAMS_CREATED = new ArrayList<>();
+    private static final List<String> STREAM_LOC_IDS = new ArrayList<>();
+
+    @BeforeAll
+    public static void setup() throws SQLException {
+        String testLoc = "StreamLoc321"; // match the stream location name in the json file
+        createLocation(testLoc, true, OFFICE_ID, "STREAM_LOCATION");
+        STREAM_LOC_IDS.add(testLoc);
+        createAndStoreTestStream("ImOnThisStream2");
+    }
+
+    static void createAndStoreTestStream(String testLoc) throws SQLException {
+        createLocation(testLoc, true, OFFICE_ID, "STREAM");
+        CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+        db.connection(c -> {
+            StreamDao streamDao = new StreamDao(getDslContext(c, OFFICE_ID));
+            Stream streamToStore = new Stream.Builder()
+                    .withId(new CwmsId.Builder()
+                            .withOfficeId(OFFICE_ID)
+                            .withName(testLoc)
+                            .build())
+                    .withLength(100.0)
+                    .withLengthUnits("km")
+                    .build();
+            STREAMS_CREATED.add(streamToStore);
+            streamDao.storeStream(streamToStore, false);
+        }, CwmsDataApiSetupCallback.getWebUser());
+    }
+
+    @AfterAll
+    public static void tearDown() {
+        for (Stream stream : STREAMS_CREATED) {
+            try {
+                CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+                db.connection(c -> {
+                    StreamDao streamDao = new StreamDao(getDslContext(c, OFFICE_ID));
+                    try {
+                        streamDao.deleteStream(stream.getId().getOfficeId(), stream.getId().getName(), DeleteRule.DELETE_ALL);
+                    } catch (Exception e) {
+                        // ignore
```

Comment:
You may want to log these exceptions with a logger, just so we can be aware when this fails to delete a stream.

Reply:
I added a logging at info level to display a failure